src/modules/auth/containers/LoginCard.js

```diff
@@ -235,6 +246,7 @@ function LoginCard(props) {
 				"danger",
 				t("LoginCard|The provided information is likely incorrect. The login has failed"), 30
 			);
+			document.getElementById("username").focus();
```

Comment:
@queuing4oranges Is this even necessary? After prologue you dont need to focus/autofocus to username anymore I suppose